Leanplum-SDK/Classes/Leanplum.m

```diff
@@ -667,6 +667,16 @@ + (NSString *)platform
 + (void)startWithUserId:(NSString *)userId
          userAttributes:(NSDictionary *)attributes
         responseHandler:(LeanplumStartBlock)startResponse
+{   [self startWithUserId:userId
+           userAttributes:attributes
+          responseHandler:startResponse
+includeContentAssignments:NO];
+}
+
++ (void)startWithUserId:(NSString *)userId
+         userAttributes:(NSDictionary *)attributes
+        responseHandler:(LeanplumStartBlock)startResponse
+includeContentAssignments:(BOOL)includeContentAssignments
```

Comment:
Is this feature for one customer or for everyone? 
I don't feel comfortable making a new Start just for enabling this flag.